volapi/chat.py

```diff
@@ -17,26 +18,29 @@ class Roles(Enum):
 
     @classmethod
     def from_options(cls, options):
+        role_list = []
```

Comment:
Replace list with `set()`. This is a minor thing, but since all roles are unique, (you can't be pro twice for example) set is the most suitable container for this property.
```suggestion
        role_set = set()
```

Reply:
Did This ✔️